### PR TITLE
Remove Cargo.lock and update dependency micro versions

### PR DESCRIPTION
**Pull request by samdealy**
Add micro versions for dependencies.

### CODE REVIEW

**Review by jcape — CHANGES_REQUESTED** [review]
This is actually downgrading quote, is something specifying `quote =  "=1.0.9"` somewhere?

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 .cargo
 target/
+Cargo.lock
+.idea/
```
Comment (jcape) [inline .gitignore]:
```suggestion
```

**Review by jcape — CHANGES_REQUESTED** [review]

**Comment by samdealy** [comment]
> This is actually downgrading quote, is something specifying `quote = "=1.0.9"` somewhere?

This is no longer the case (for posterity).

**Review by jcape — APPROVED** [review]

**Review by nick-mobilecoin — APPROVED** [review]

**Review by jcape — CHANGES_REQUESTED** [review]
Please remove `.idea` from `.gitignore`, and use the global ignore.

**Review by jcape — APPROVED** [review]

**Pull request merged**